plonky2x/core/src/frontend/hash/sha/sha256/curta.rs

```diff
@@ -114,13 +113,13 @@ impl<L: PlonkParameters<D>, const D: usize> CircuitBuilder<L, D> {
         &mut self,
         input: &[ByteVariable],
         length: U32Variable,
-        last_chunk: U32Variable,
     ) -> Bytes32Variable {
         assert_eq!(
             input.len() % 64,
             0,
             "input length should be a multiple of 64"
         );
+        let last_chunk = self.compute_sha256_last_chunk(length);
```

Comment:
nit: You refer to `last_chunk` as `chunk_index` in `SHARequest` enum, update `last_chunk` to `chunk_index`. Also, I think `final_chunk_index` might be a bit more clear.